docs/content/latest/explore/ysql-language-features/gin.md

```diff
@@ -347,11 +345,6 @@ SubDocKey(DocKey([], ["\x05one", EncodedSubDocKey(DocKey(0xfca0, [3], []), [])])
 SubDocKey(DocKey([], ["\x05thing", EncodedSubDocKey(DocKey(0x9eaf, [4], []), [])]), [SystemColumnId(0); HT{ physical: 1636678107997627 w: 1 }]) -> null; intent doc ht: HT{ physical: 1636678107961628 }
 ```
 
-## Roadmap
-
-GIN indexes are still in progress: see the GIN roadmap tracking [GitHub issue][github-roadmap] for more details.
```

Comment:
Why are you taking this out? It seems important to let the reader know that this is an incomplete feature, and give them a link to the tracking ticket for it. We do this for other features.

Reply:
This is just Explore section. Also all this should go in Github Master ticket.